@here/harp-examples/package.json

```diff
@@ -15,7 +15,7 @@
     "scripts": {
         "test": "exit 0",
         "prebuild": "node scripts/postinstall.js",
-        "build": "webpack",
+        "build": "webpack -p",
```

Comment:
This makes production build the default. Not sure we want this.
I think, that main improvement we want from this change is that bundles deployed in `harp.gl` are minified.
So, i guess we need adaptation of apropriate deploy scripts (in travis-ci.yml) to set `NODE_ENV=production` and use this variable in `webpack.config`s.

Reply:
I changed the package scripts back to building without the production flag. The production environment is now set in the Travis deploy script. I also fixed the Webpack config of the examples to copy the right `harp.gl` bundle depending on the configured environment.